lib/utils/testLauncherHelper.js

```diff
@@ -88,13 +88,13 @@ function followChildProcess(child, callback) {
 		input: child.stdout,
 		terminal: false,
 	}).on('line', line => {
-		callback && callback(line.toString());
```

Comment:
you can use callback without additional checking if will write at the function beginning: callback  = callback || () => {},

Reply:
@JeKachan I think it's the same, just a matter of taste.

What should be added, however, is jsdoc mark that parameter is optional:
` * @param {Function} [callback] callback function`